modules/Stories/Stories.tsx

```diff
@@ -9,7 +9,7 @@ type Props = {
 const Stories: FunctionComponent<Props> = ({ stories }) => (
     <div>
         {stories.map((story) => (
-            <Link key={story.id} href={`/story/${story.id}`} passHref>
+            <Link key={story.id} href={story.slug} passHref>
```

Comment:
I guess it should be `/${story.slug}` with a slash?

Reply:
`story.slug` works too, it will redirect properly